server/data/restClient.ts

```diff
@@ -155,12 +155,10 @@ export default class RestClient {
             logger.info(`Error caught for get stream ${path}`)
             reject(error)
           } else if (response) {
-            const s = new Readable()
-            // eslint-disable-next-line no-underscore-dangle,@typescript-eslint/no-empty-function
-            s._read = () => {}
-            s.push(response.body)
-            s.push(null)
-            resolve(s)
+            const streamedResponse = Readable.from(response.body)
+            streamedResponse.push(response.body)
+            streamedResponse.push(null)
```

Comment:
Not sure what pushing these does, but I would have assumed using `Readable.from(response.body)` would mean you dont have to push `response.body` again?

Reply:
https://stackoverflow.com/questions/12755997/how-to-create-streams-from-string-in-node-js#answer-59638132

Reply:
I spent some time on this and without the push it did not return the image. I agree its odd, but slightly better than it was and now has a test around it.